jwst/lib/set_telescope_pointing.py

```diff
@@ -914,9 +930,13 @@ def calc_wcs_tr_202111(transforms: Transforms):
 def calc_transforms(t_pars: TransformParameters):
     """Calculate transforms  which determine reference point celestial WCS
 
-    Given the spacecraft pointing parameters and the
-    aperture-specific SIAF, calculate all the transforms
-    necessary to produce WCS information.
+    This implements Eq. 3 from Technical Report JWST-STScI-003222, SM-12. Rev. C, 2021-11
+    From Section 3:
+
+    The Direction Cosine Matrix (DCM) that provides the transformation of a
+    unit pointing vector defined in inertial frame (ECI J2000) coordinates to a
+    unit vector defined in the science aperture Ideal frame coordinates is
```

Comment:
This says going to Ideal coordinates. However, for the older methods, it's actually ICS coordinates. I'd suggest removing all the older methods to avoid any confusion.

It'd also be easier to understand the code if the attribute names of transforms with "siaf" be changes to idl.

Reply:
Will do the removal of old code in another PR.